scripts/extract_validation_messages.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import yaml, os, shutil
+
+dir = os.path.dirname(__file__)
+
+if os.path.exists('{}/validation_messages'.format(dir)):
+    shutil.rmtree('{}/validation_messages'.format(dir))
+
+os.makedirs('{}/validation_messages'.format(dir))
```

Comment:
@risicle Yep, just moved it all down, as well as renaming dir to something more better.